windows/devices/uwb/include/windows/devices/uwb/UwbConnector.hxx

```diff
@@ -226,15 +221,52 @@ private:
     std::vector<std::shared_ptr<::uwb::UwbRegisteredDeviceEventCallbacks>>
     GetResolvedDeviceEventCallbacks();
 
+    /**
+     * @brief Internal helper function to insert the callback into the hashmap of ids to callbacks
+     * You MUST have grabbed the m_eventCallbacksGate mutex prior to calling this.
+     *
+     * @param callback
+     * @return std::weak_ptr<::uwb::RegisteredCallbackToken>

```

Comment:
The function returns `std::shared_ptr`, so this comment should be updated

Reply:
not sure what you're referring to here, as the function returns a weak_ptr

Reply:
Seems the return type should be `std::weak_ptr<::uwb::RegisteredCallbackToken>` since the public API function (correctly) returns as-such.